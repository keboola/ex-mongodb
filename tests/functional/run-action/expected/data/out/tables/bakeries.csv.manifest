{"incremental":true,"columns":["id","name"],"primary_key":["id"],"column_metadata":{"id":[{"key":"KBC.datatype.nullable","value":true}],"name":[{"key":"KBC.datatype.nullable","value":true}]}}