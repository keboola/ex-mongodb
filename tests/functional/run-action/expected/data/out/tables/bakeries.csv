"5716054cee6e764c94fa9f31","1617-A National Bakery"
"5716054cee6e764c94fad66f","Jacqueline'S Bakery Restaurant"
"5716054bee6e764c94fa8c13","La Nueva Giralda Bakery"
"5716054bee6e764c94fa841e","Nacional Bakery #1"
"5716054bee6e764c94fa8ff6","National Bakery"
"5716054cee6e764c94faa105","National Bakery"
